src/components/AgentInput/index.js

```diff
@@ -23,10 +24,11 @@ class AgentInput extends Component {
 
   render() {
     return (
-      <form>
-        <input name="hint" type="text" value={this.state.hint} onChange={this.handleChange} />
-        <input name="relWords" type="number" value={this.state.relWords} onChange={this.handleChange} />
-        <input type="submit" onClick={this.handleSubmit} />
+      <form className="AgentInput">
+        <ConversationsList name="hint" type="text" value={this.state.hint} onChange={this.handleChange} />
```

Comment:
I'm definitely going to have to see how these websockets play out. I'm still really confused at the moment 🤔